DfE.FindInformationAcademiesTrusts/assets/sass/_resets.scss

```diff
@@ -8,3 +8,9 @@ dd {
 ul > li {
   margin-bottom: 0;
 }
+
+.govuk-breadcrumbs > ol,
+ul,
+li {
+  @include govuk-font-size(16);
```

Comment:
This affects every `ul` and every `li` including the main navigation component and is applied after the gov-uk import so it won't be overridden. 
This should be more specific and also located nearer to the problem - so if it is the DfE design system causing the issue then we could put it in `DfE.FindInformationAcademiesTrusts/assets/sass/imports/_dfe-frontend.scss`